src/playwrightTest.ts

```diff
@@ -149,17 +151,22 @@ export class PlaywrightTest {
       '--repeat-each', '1',
       '--retries', '0',
     ];
-    if (this._isUnderTest)
+    if (this._isUnderTest || !!this._browserServerWS)
```

Comment:
Should we check that playwright versions from `config` and `runBrowserServer` match and error otherwise?

Reply:
That would mean that we have a multi-workspace configuration in VS Code or a monorepo where different projects have different versions of Playwright. I'd say ignore this edge case for now? I should add a minimum version check for run-server though, I'll do that.